falco-exporter/templates/role.yaml

```diff
@@ -0,0 +1,22 @@
+{{- if .Values.podSecurityPolicy.create -}}
+kind: Role
+apiVersion: rbac.authorization.k8s.io/v1beta1
```

Comment:
Sure, I will update it like that. Only for the Role & Rolebinding or also for the PodSecurityPolicy, Daemonset, Server, GrafanaDashboard ConfigMap, Secrets and ServiceAccount? Basically all except "ServiceMonitor" (since it's a CRD)?

Perhaps stupid question, but where does `rbac.apiVersion` come from? Do you have a link to the documentation somewhere?

Reply:
Ah got it! Its a custom helper: https://github.com/falcosecurity/charts/blob/7e511c7f60fd5a591a8f13a97c84ba36b87874eb/falco/templates/_helpers.tpl#L77-L86